src/test/fixtures/utils.js

```diff
@@ -341,8 +341,61 @@ export function fireFullContextMenu(
  * And addition Julien's review comments here:
  * https://github.com/firefox-devtools/profiler/pull/2842/commits/6be20eb6eafca56644b1d55010cc1824a1d03695#r501061693
  */
-export function fireFullKeyPress(element: HTMLElement, options?: *) {
-  fireEvent.keyDown(element, options);
-  fireEvent.keyPress(element, options);
-  fireEvent.keyUp(element, options);
+export function fireFullKeyPress(
+  element: HTMLElement,
+  options: { key: string, ... }
+) {
+  // Note the key is lowercase here.
+  const codes: { [key: string]: number } = {
```

Comment:
nit: please add a comment saying it's valid for QWERTY keyboards. (and that's OK for tests, I guess)